tests/test_eval.py

```diff
@@ -40,10 +40,7 @@ def common_test(extra_args: List[str]) -> None:
         assert recurse_drv["attr"] == "recurse.drvB"
         assert recurse_drv["name"] == "drvB"
 
-        recurse_recurse_bool = results[3]
-        assert "error" in recurse_recurse_bool
```

Comment:
Don't know why we wanted an error here? Now I just make `nix-eval-jobs` ignore everything that is not a derivation, which seems more intuitive to me.